package/rmt-server.spec

```diff
@@ -225,6 +226,7 @@ getent passwd %{rmt_user} >/dev/null || \
 %service_add_post rmt-server.target rmt-server.service rmt-server-migration.service rmt-server-mirror.service rmt-server-sync.service
 cd /usr/share/rmt && runuser -u %{rmt_user} -g %{rmt_group} -- bin/rails secrets:setup >/dev/null
 cd /usr/share/rmt && runuser -u %{rmt_user} -g %{rmt_group} -- bin/rails runner -e production "Rails::Secrets.write({'production' => {'secret_key_base' => SecureRandom.hex(64)}}.to_yaml)"
+echo "The installation completed. Please run yast2-rmt to configure the package properly" >> /dev/stdout
```

Comment:
please check the content of the message

Reply:
@Sergeykot could you also please fix #165 while you are at it? (I think you just need to redirect `Rails::Secrets.write` line to `/dev/null`)

Reply:
done